ext/dba/config.m4

```diff
@@ -135,33 +135,21 @@ dnl
 
 dnl QDBM
 if test "$PHP_QDBM" != "no"; then
-  PHP_DBA_STD_BEGIN
-  for i in $PHP_QDBM /usr/local /usr; do
-    if test -f "$i/include/depot.h"; then
-      THIS_PREFIX=$i
-      THIS_INCLUDE=$i/include/depot.h
-      break
-    fi
-  done
-
-  if test -n "$THIS_INCLUDE"; then
-    for LIB in qdbm; do
-      PHP_CHECK_LIBRARY($LIB, dpopen, [
-        AC_DEFINE_UNQUOTED(QDBM_INCLUDE_FILE, "$THIS_INCLUDE", [ ])
-        AC_DEFINE(DBA_QDBM, 1, [ ])
-        THIS_LIBS=$LIB
-      ], [], [-L$THIS_PREFIX/$PHP_LIBDIR])
-      if test -n "$THIS_LIBS"; then
-        break
-      fi
-    done
-  fi
-
-  PHP_DBA_STD_ASSIGN
-  PHP_DBA_STD_CHECK
-  PHP_DBA_STD_ATTACH
+  PKG_CHECK_MODULES([QDBM], [qdbm])
+
+  PHP_EVAL_INCLINE($QDBM_CFLAGS)
+  PHP_EVAL_LIBLINE($QDBM_LIBS, DBA_SHARED_LIBADD)
+
+  PHP_CHECK_LIBRARY(qdbm, dpopen,
```

Comment:
You can drop the CHECK_LIBRARY call and directly execute those AC_DEFINE's. Same below. If pkg-config finds a package, we assume that it's working correctly and don't perform additional library checks.